river/compose/union.py

```diff
@@ -118,6 +120,35 @@ class TransformerUnion(base.Transformer):
     ...     ('# by place', count)
     ... )
 
+    Mini-batch example:
+
+    >>> X = [
+    ... {"place": 2, "revenue": 42},
```

Comment:
Add 4 spaces before each `dict`

Reply:
Btw do you know of any easy way to put code into these doctests? Like less manual/error-prone

Reply:
I think it's possible to configure black to lint doctests, but I haven't looked into it